srcpkgs/python3-watchdog/template

```diff
@@ -1,17 +1,19 @@
 # Template file for 'python3-watchdog'
 pkgname=python3-watchdog
-version=2.1.6
-revision=2
+version=2.2.1
+revision=1
 build_style=python3-module
+make_check_args="--deselect tests/test_inotify_buffer.py::test_unmount_watched_directory_filesystem"
 hostmakedepends="python3-setuptools"
-depends="python3-yaml python3-argh python3-pathtools"
+depends="python3-yaml"
 checkdepends="python3-pytest $depends"
 short_desc="Python3 API and shell utilities to monitor file system events"
 maintainer="Maurizio Porrato <maurizio.porrato@gmail.com>"
 license="Apache-2.0"
-homepage="http://github.com/gorakhargosh/watchdog"
+homepage="https://pypi.org/project/watchdog/"
```

Comment:
let's use github for homepage, update-check looks at the $distfiles and $homepage URLs, so having two separate places is nice.

Reply:
Oh! I actually have been actively changing quite a few python package's homepage so that the daily auto update checker doesn't hit github too hard. So many false positives in the failed update checker log due to rate limiting from github.

Reply:
That's a good reason to use pypi, I know that @classabbyamp was interested in using the github API for the bulk update checker which could solve this.